### DISCUSSION TITLE
Welcome to scRNAtoolVis Discussions!

### DISCUSSION THREAD

**Discussion by junjunlab** [junjunlab/scRNAtoolVis#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Zicong1107** [comment]
For jjVolcano, can I use multiple different colors to mark different sets of markers?

**Reply by junjunlab** [reply]
Nop, maybe you can re-write the source code to draw plot.
